internal/pkg/cgroups/config_linux.go

```diff
@@ -191,13 +191,3 @@ func LoadConfig(confPath string) (config Config, err error) {
 	err = toml.Unmarshal(b, &config)
 	return
 }
-
-// PutConfig takes the content of a CgroupsConfig struct and Marshals it to file
-func PutConfig(config Config, confPath string) (err error) {
```

Comment:
We should leave this for completeness of the `Get/Set` style api for this struct. This is actually used by this example plugin: https://github.com/apptainer/apptainer/blob/main/examples/plugins/config-plugin/main_linux.go#L54

Reply:
will preserve